sulu/sulu/1.6/src/SuluKernel.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App;
+
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Routing\RouteCollectionBuilder;
+
+class SuluKernel extends Kernel
+{
+    /**
+     * @var string string
+     */
+    private $context;
+
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    public function __construct(string $environment, $debug)
+    {
+        parent::__construct($environment, $debug);
+
+        $this->context = $this->detectContext();
+
+        $configurations = require dirname(__DIR__) . '/config/contexts.php';
```

Comment:
should use getConfigDir() I guess?

Reply:
correct :) but this is more like a working prototype! we will refactor it if we are finished. i will ping you then